HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 History
 =======
 
+0.2.4 (XXXX-XX-XX)
+------------------
+
+* Renaming estimators module to Regression
```

Comment:
Changer en "Renaming `estimators.py` module to `regression.py`

Reply:
c'est changé